### ISSUE TITLE
oGridAddLayout broken?

### ISSUE THREAD

**Issue by wheresrhys** [Financial-Times/o-grid#111]
The following works
```
$o-grid-is-silent: false;
$o-grid-layouts: (
	S:   490px,
	M:   740px,
	L:   980px,
	XL:  1220px,
	XXL: 1600px
);
@import 'o-grid/main';
```

But this (which *should* do the same, only without having to redefine the default sizes) appears to do nothing
```
@import 'o-grid/main';
@include oGridAddLayout(
     $layout-name: XXL,
     $layout-width: 1600px
);
@include oGridSurfaceCurrentLayout;
@include oGridGenerate;
```

**Comment by wheresrhys** [comment]
As an aside, it'd be great if the following shorthand for the above were supported
```
@import 'o-grid/main';
@include oGridAddLayout(
     $layout-name: XXL,
     $layout-width: 1600px,
     $generate: true
);
```

**Comment by kaelig** [comment]
Looks like adding a large layout (larger than the rest) fails.

Interestingly enough, there is no unit test for this use case.

A good first step would be to add a test case here:
https://github.com/Financial-Times/o-grid/blob/master/test/

**Comment by AlbertoElias** [comment]
Fixed in #119

**Comment by kaelig** [comment]
Well done!
